.gitignore

```diff
@@ -33,3 +33,7 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+public/sw.js
+public/sw.js.map
+public/workbox-e43f5367.js
```

Comment:
Are these specific? If so should update to `public/workbox-*.js`

Reply:
These weren't specific. @kutagorn @god-hephaestus make sure your gitignore is updated as of 2c7eca4b404f56a874908aaac848c0dfa6e58f43 in the [`unique-routes`](https://github.com/ozgurozbek/teothe/tree/unique-routes) branch.